package.json

```diff
@@ -5,7 +5,7 @@
   "dependencies": {
     "babel-runtime": "^6.20.0",
     "bluebird": "~2.9.24",
-    "mobify-progressive-app-sdk": "~0.18.0"
+    "mobify-progressive-app-sdk": "1.0.0-beta4"
```

Comment:
Yea but I haven't released it yet so it wouldn't build if I had `rc1`. There's no changes from beta4 to rc1 so I was planning on just updating it when I release beta1